tests/sentry/api/test_client.py

```diff
@@ -0,0 +1,174 @@
+from unittest.mock import MagicMock, Mock, patch
+
+from django.http import JsonResponse
+
+from sentry.api.client import ApiClient
+from sentry.testutils.cases import TestCase
+
+
+class ClientParameterHandlingTest(TestCase):
+    @patch("sentry.api.client.resolve")
+    @patch("sentry.api.client.APIRequestFactory")
+    @patch("sentry.api.client.force_authenticate")
+    def test_list_parameters_use_setlist(self, mock_force_auth, mock_factory_class, mock_resolve):
```

Comment:
might it be simpler just to mock the final `callback` and make sure the request URL is correct?